fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -456,8 +456,7 @@ class HomeFragment : Fragment() {
             ).also { it.build() }
 
             BottomToolbarContainerView(
-                context = requireContext(),
```

Comment:
nit: Don't think we need to remove this context parameter.

Reply:
addressed the nit